lib/bundle/commands/cleanup.rb

```diff
@@ -151,7 +151,7 @@ def vscode_extensions_to_uninstall(global: false, file: nil)
         return [].freeze if kept_extensions.empty?
 
         current_extensions = Bundle::VscodeExtensionDumper.extensions
-        current_extensions - kept_extensions
+        current_extensions.map(&:downcase) - kept_extensions.map(&:downcase)
```

Comment:
I think it might make more sense to add `.map(&:downcase)` to `kept_extensions` where it's originally queried above.

Reply:
```suggestion
        current_extensions - kept_extensions
```

I think it might make more sense to add `.map(&:downcase)` to `kept_extensions` where it's originally queried above.

Reply:
done